src/foam/nanos/crunch/ui/MinMaxCapabilityWizardlet.js

```diff
@@ -264,16 +262,55 @@ foam.CLASS({
 
 
         // Update lifted choices based on their availability
-        let newSelectedData = [...this.selectedData];
+        let newSelectedData = [...this.data.selectedData];
         for ( const w of liftedWizardlets ) {
           if ( w.isAvailable ) newSelectedData.push(w.capability);
           else foam.Array.remove(newSelectedData, w.capability);
         }
-        this.selectedData = foam.Array.unique(newSelectedData);
+        this.data.selectedData = foam.Array.unique(newSelectedData);
       }
       const slots = liftedWizardlets.map(w => w.isAvailable$);
       this.ArraySlot.create({ slots }).sub(updated);
       this.isAvailable$.sub(updated);
-    }
+    },
+    function getPrerequisiteAvailabilitySlot(prereqWizardlet){
+      var slot = foam.core.SimpleSlot.create();
+
+      this.data.selectedData$.sub(
+        ()=> slot.set(this.isPrereqSelected(prereqWizardlet))
```

Comment:
If I set `isAvailable` on a prerequisite wizardlet does `selectedData` update as well?

Reply:
We would need to build that two-way binding between PrerequisiteAwareWizardlet and the PrerequisiteWizardlet in. Current it's one way (from PrerequisiteAwareWizardlet -> PrerequisiteWizardlet).

Could potentially see a feedback loop occur though.